aws-ecs/20-bastion.tf

```diff
@@ -13,33 +13,33 @@ data "aws_ami" "amazon_linux_2" {
 
 resource "aws_security_group" "bastion" {
   count  = var.create_bastion ? 1 : 0
-  name   = "bastion-sg"
+  name   = "${var.deployment_id}-bastion-sg"
```

Comment:
Same comment about the redundant `-sg` (I will stop commenting that now.. I guess the comment applies to all resources names)